### DISCUSSION TITLE
寶鐘瑪琳真可愛

### DISCUSSION THREAD

**Discussion by LeoTsai999** [1101-GS4719-C/Python-Community-GS4719-C#2257] [General]
寶鐘瑪琳是hololive三期生的成員。我認為我們應該和寶鐘瑪琳一樣努力工作，藉此實現買下一艘船的目標！
我個人建議大家可以可以嘗試和寶鐘瑪琳學習一些航海知識，也可以學習一些直播的技巧，增進社交能力！
更具體來說，我建議大家可以前往寶鐘瑪琳的youtube頻道https://www.youtube.com/@HoushouMarine 訂閱並準時觀看直播。
另一個建議是，我們可以加入寶鐘瑪琳的頻道會員，這樣便可以觀看會員限定影片，獲取更多知識！

**Comment by Karsta0216** [comment]
但你的頭像是koyori...
我認為獅白牡丹的頻道風格給人一種很舒服的感覺，非常推薦大家去觀看他的直播以及烤肉！
